### ISSUE TITLE
Need your help about the training hyperparameters

### ISSUE THREAD

**Issue by gavin1332** [ecnu-sea/SEA#7]
Thanks for your inspirational project! We are try to follow up on your work.

Could you share the training hyperparameters including the learning rate, batch size, and the optimizer?

**Comment by heroding77** [comment]
Hello,

Thank you for following up on our work! We used the Llama-Factory framework for training. The script below includes all the parameters we used:

```
# model
model_name_or_path: mistral/model/Mistral-7B-Instruct-v0.2

# method
stage: sft
do_train: true
finetuning_type: lora
lora_target: all
lora_rank: 8
lora_alpha: 16
lora_dropout: 0.1
use_dora: true
flash_attn: fa2

# ddp
ddp_timeout: 180000000
deepspeed: examples/deepspeed/ds_z3_config.json

# dataset
dataset: sea_instructionv3_Nv1_Iv3_mmd_random
template: mistral
cutoff_len: 32768
max_samples: 10000
val_size: 0.0
overwrite_cache: true
preprocessing_num_workers: 16

# output
output_dir: saves/Mistral/lora/sft_mistral_instructionv3_random
logging_steps: 4
save_steps: 100
plot_loss: true
overwrite_output_dir: true

# train
per_device_train_batch_size: 1
gradient_accumulation_steps: 8
learning_rate: 0.0001
num_train_epochs: 4.0
lr_scheduler_type: cosine
warmup_steps: 0
fp16: true
```

I hope this information is helpful to you. If you have any further requests, please don’t hesitate to let me know!

**Comment by gavin1332** [comment]
Thank you for your detailed response. ^_^
